posthog/async_migrations/test/test_0010_move_old_partitions.py

```diff
@@ -82,4 +82,4 @@ def test_completes_successfully(self):
         self.assertTrue(run_migration())
 
         # this test is not very helpful, but we will at least catch if this changes
-        self.assertEqual(len(MIGRATION_DEFINITION.operations), FuzzyInt(5, 7))
```

Comment:
I am not sure why those were failing, but they seem related to the new entries on `clickhouse/schema.py`